src/index.tsx

```diff
@@ -1,28 +1,59 @@
-import { render } from 'react-dom';
-
+import '@contentful/forma-36-fcss/dist/styles.css';
+import '@contentful/forma-36-react-components/dist/styles.css';
+import '@contentful/forma-36-tokens/dist/css/index.css';
 import {
   AppExtensionSDK,
-  FieldExtensionSDK,
   DialogExtensionSDK,
+  FieldExtensionSDK,
   init,
   locations,
 } from 'contentful-ui-extensions-sdk';
-import '@contentful/forma-36-react-components/dist/styles.css';
-import '@contentful/forma-36-fcss/dist/styles.css';
-import '@contentful/forma-36-tokens/dist/css/index.css';
+import { render } from 'react-dom';
 
 import Config from './components/ConfigScreen';
-import Field from './components/Field/';
 import Dialog from './components/Dialog/';
+import Field from './components/Field/';
```

Comment:
Do you mind disabling code formatting on save? Each of the diffs on these PRs are getting buffed with import statements being shuffled around 😅